src/config.js

```diff
@@ -11,7 +11,8 @@ class Config {
       language,
       host,
       storage,
-      custom_fetch
+      custom_fetch,
+      authenticate
```

Comment:
Maybe we could name this `custom_authenticator` to match the existing naming and to show that it is optional?